packages/amplication-data-service-generator/src/tests/__snapshots__/create-data-service-without-graphql.spec.ts.snap

```diff
@@ -14482,6 +14376,112 @@ export class StringNullableFilter {
   @Type(() => String)
   not?: string;
 }
+",
+  "undefined/App.tsx": "import React, { useEffect, useState } from \\"react\\";
```

Comment:
something is wrong in the snapshot data - please note the URL starts with `undefined`
```suggestion
  "admin-ui/App.tsx": "import React, { useEffect, useState } from \\"react\\";
```